### DISCUSSION TITLE
Linux Build?

### DISCUSSION THREAD

**Discussion by rathboma** [johnlindquist/kit#668] [Q&A]
This looks great, but everyone know Linux is the best os :-P.

Seriously though, how can I help you make a Linux build? Is there a lot of platform specific code?

**Comment by johnlindquist** [comment]
I promise I'm trying 😅

<img width="1132" alt="CleanShot 2021-12-22 at 20 47 27@2x" src="https://user-images.githubusercontent.com/36073/147185299-817d2dbb-462d-446c-8f74-a3e47de3560a.png">

**Reply by rathboma** [reply]
Nice!

**Reply by johnlindquist** [reply]
It's looking promising. Still a few more install bugs to sort out...


https://user-images.githubusercontent.com/36073/147198203-ae3eea85-0bf0-4ff2-84a2-e388aebe879c.mp4

**Reply by eboye** [reply]
Cool! Looking forward to this :)

But Arch based distro here, so don't forget to support us as well.

**Reply by pablomuro** [reply]
@johnlindquist can you create a little video or steps on how to run/setup/create the installer of kit on Linux?
Your idea is amazing and if I'm capable of, I would like to try to make kit run on Linux.

**Reply by canvural** [reply]
@pablomuro Did you try the AppImage? It works very well for me on Ubuntu. You don't need to build it yourself. Check the releases and download the AppImage

**Reply by pablomuro** [reply]
@canvural thnks, I was in the Kit core repository I didn't check the Kitapp
I think that could be on the Readme.md of the core repo as a note that the Kitapp has an AppImage for the Linux

**Reply by johnlindquist** [reply]
@pablomuro I don't have the time to support Linux builds, so I intentionally never talk about it. 

Maybe once V2 is out next year, I can settle down enough to give it some focus, but until then it will remain in the state of: "If you can find it and it works for you"

**Reply by kseikyo** [reply]
I mean, I have used arch for multiple years now, and I understand it's hard to maintain software. If linux users want support for something, they build it. That's how it has been in the past and probably will be in the future.

**Comment by ishanray** [comment]
Would be nice to have a https://canny.io/ board for this so we can +1 these issues? Would it be helpful @johnlindquist?

**Reply by johnlindquist** [reply]
Unfortunately, no one has stepped up to help with Linux. I achieved the goal of "it works on Linux" (meaning "I tested in an Ubuntu vm"), but I just can't commit the time and effort to all the details based on distros, etc that would be required. It's hard enough keeping up with OSX removing python in 12.3 😅.. 

Once I sort out this next batch of bugs and get the build out, I can send out a request for help with Linux.
